cli/local/api.go

```diff
@@ -88,6 +111,13 @@ func UpdateAPI(apiConfig *userconfig.API, configPath string, projectID string, a
 	}
 
 	if prevAPISpec == nil && len(prevAPIContainers) == 0 {
+		if encounteredVersionMismatch {
+			return newAPISpec, fmt.Sprintf(
+				"re-creating api %s with current CLI version %s",
```

Comment:
nit: `re-creating` -> `re-deploying`

Reply:
Done. What do you think of doing it for `creating` -> `deploying` too?

Reply:
Good point, instead of `re-deploying` just `creating api %s with current CLI version %s` is fine

Reply:
Done.